### ISSUE TITLE
cannot find rule: use-errors-new

### ISSUE THREAD

**Issue by comerc** [mgechev/revive#1266]
**Describe the bug**

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
# flags

revive ...
```

```toml
# config file
```

**Expected behavior**

**Logs**

**Desktop (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04]
 - Version of Go

**Additional context**

**Comment by alexandear** [comment]
@comerc Thank you for reporting the issue! However, could you provide more details to help us better understand the problem? Specifically:

1. The contents of your configuration file (or at least the relevant parts).
2. The exact output/error message you received.
3. The version of Go you are using (`go version`).

**Comment by alexandear** [comment]
Related to #1265
